### PR TITLE
challenge update

### PR DESCRIPTION
**Pull request by ebbauer**
please check it out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +16,17 @@ This entity should contain the following fields: id, company name and zip code.
 suport file: q1_catalog.csv
 
 
-**2 - An API to integrate data using a database**
+### 2 - An API to integrate data using a database
 
-Yawoen company started to capture website data from another source and want to integrate it with the entity you've just created on the database.
+Yawoen company started to capture website data from another source and want to integrate it with the entity you've just created on the database. When possible, it's mandatory that the data are merged.
```
Comment (kamilash) [inline README.md]: Suggestion: It's mandatory the data to be merged, when the requirements for that are met.

File: README.md [hunk 2]
```diff
@@ -64,12 +70,11 @@ Ouput example:
 - Document your API if you are willing to.
 
 
-### Deliverable
+## Deliverable
 
 
-- A git repo hosted wherever you like.
+- It would be REALLY nice if it was hosted in a git repo of your own. You can create a new empty project, create a branch and Merge Request it to the master branch. Provide the MR URL for us. BUT if you'd rather, just compress this directory and send it back to us.
```
Comment (kamilash) [inline README.md]: Maybe better to use "Pull request" and "PR", since the code is on github.

**Comment by HenioJR** [comment]
Top!

File: README.md (hunk 2 above)
Comment (HenioJR) [reply README.md]: Yes, github use PULL REQUEST

File: README.md (hunk 1 above)
Comment (ebbauer) [reply README.md]: nice, thanks

File: README.md (hunk 2 above)
Comment (ebbauer) [reply README.md]: good one

**Review by kamilash — APPROVED** [review]
LGTM!

**Pull request merged**